doc/api/fs.md

```diff
@@ -3249,22 +3249,31 @@ changes:
 * `callback` {Function}
   * `err` {Error}
 
-Asynchronous symlink(2). No arguments other than a possible exception are given
-to the completion callback. The `type` argument is only available on Windows
+Asynchronous symlink(2) which creates the link called `path` pointing to `target`.
+No arguments other than a possible exception are given to the completion callback. 
+
+The `type` argument is only available on Windows
 and ignored on other platforms. It can be set to `'dir'`, `'file'`, or
 `'junction'`. If the `type` argument is not set, Node will autodetect `target`
 type and use `'file'` or `'dir'`. If the `target` does not exist, `'file'` will
 be used. Windows junction points require the destination path to be absolute.
 When using `'junction'`, the `target` argument will automatically be normalized
 to absolute path.
 
-Here is an example below:
+Note that relative targets are relative to the link’s parent directory.
 
 ```js
-fs.symlink('./foo', './new-port', callback);
+fs.symlink('./mew', './example/mewtwo', callback);
 ```
 
-It creates a symbolic link named "new-port" that points to "foo".
+creates a symbolic link `mewtwo` in the `example` which points to `mew` in the same directory:
```

Comment:
Can you make sure to wrap lines at 80 characters?

Reply:
Sure. Any hints on how to do this easily? :) WebStorm does not seem to support reflowing paragraphs (or I did not find it).

Reply:
You can always look whether the line length roughly matches the rest of the file. If it’s too much trouble, I think it shouldn’t be an issue for the person who merges this to take care of the linting issues.

Reply:
Found an easy solution, [vim can do it](https://vim.fandom.com/wiki/Automatic_formatting_of_paragraphs) :)